platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2349,7 +2349,7 @@
 		DA25D5BC1CCD9EDE00607828 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				INFOPLIST_FILE = framework/Settings.bundle/Root.plist;
+				INFOPLIST_FILE = "framework/Info-static.plist";
```

Comment:
Settings.bundle is an example meant to be used in any iOS application. A Settings bundle isn’t supposed to have an Info.plist, though.